web/film-club-client/src/components/FilmDetailPage.js

```diff
@@ -1,54 +1,193 @@
+								alt={`${title} poster image`}
+							/>
+						</Box>
+						<Box
+							className="trailer-box"
+							display="flex"
+							direction="row"
+							justify="around"
+							onClick={() => setShowVideo(true)}
+							focusIndicator={false}
+							margin={{ top: 'small' }}
+							pad="small"
+							border={{ size: 'small', color: 'accent-4' }}
+							round="small"
+						>
+							<Text>Watch Trailer</Text>
+							<CirclePlay size="25em" color="accent-4" />
+						</Box>
+						{showVideo && (
+							<Layer
+								modal={true}
+								responsive={true}
+								onEsc={() => setShowVideo(false)}
+								onClickOutside={() => setShowVideo(false)}
+								round="medium"
+							>
+								{videos[0] ? (
+									<iframe
+										title={title}
+										width="560"
+										height="315"
+										src={`https://www.youtube.com/embed/${videos[0].key}`}
+										frameBorder="0"
+										allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+										allowFullScreen
+									></iframe>
+								) : (
+									<Box
+										display="flex"
+										direction="column"
+										justify="center"
+										align="center"
+										pad="small"
+										responsive={true}
+									>
+										<Text size="1.25em" margin="small" color="light-3">
+											No trailer for this film yet, check back soon!
+										</Text>
+										<Box
+											border={{ size: 'small', color: 'accent-4' }}
+											color="accent-4"
+											alignSelf="center"
+											pad={{ horizontal: 'large' }}
+											align="center"
+											round="xlarge"
+											margin="small"
+											onClick={() => setShowVideo(false)}
+										>
+											close
+										</Box>
+									</Box>
+								)}
+							</Layer>
+						)}
 					</Box>
-					<Box>
-						<Heading level="2">{title}</Heading>
-						<br />
-						<Text>{overview}</Text>
+
+					<Box
+						className="info-box-wrapper"
+						display="flex"
+						direction="column"
+						justify="start"
+					>
+						<Box
+							className="info-box"
+							display="flex"
+							direction="column"
+							justify="start"
+							height="16em"
+						>
+							<Box className="title-box">
+								<Heading level="1" margin="0">
+									{title}
+								</Heading>
+								<Text size="0.9em">({releaseDate})</Text>
+								{genres[0] && <Text size="0.9em">({genres[0].name})</Text>}
+							</Box>
+							<br />
+							{tagline && (
+								<Text margin={{ top: 'xsmall', bottom: 'medium' }}>
+									{tagline}
+								</Text>
+							)}
+							<Text>{overview}</Text>
+						</Box>
 					</Box>
 				</Box>
-			) : (
-				<Heading level="2">Loading..</Heading>
-			)}
-		</Box>
-	);
+
+				<Box className="media-box">
+					{images[0] && (
+						<Box className="images-box">
+							<Image
+								src={`http://localhost:8080/v1/films/images/poster${backdropPath}?size=large`}
+								alt={`${title} poster image`}
```

Comment:
It looks like you're not even using anything from the `images` object, but you're using the `poster` endpoint with the `backdropPath`, and the `alt` text says "poster image".